### DISCUSSION TITLE
010 Editor problem

### DISCUSSION THREAD

**Discussion by malison88** [jilleb/MQB-FPA#79] [Q&A] [answered]
In every time when I try to run a template in 010 Editor I've got the following error:
![image](https://user-images.githubusercontent.com/102321737/198844180-1e1c682b-003d-4ead-a199-701d1f831eaa.png)
What did I do wrong? Thank you

**Comment by jilleb** [comment, marked as answer]
It appears you didn't download the template file but the website instead. Make sure you use the raw file.
